utility/utils.js

```diff
@@ -418,32 +418,31 @@ class Utils {
         teamAuthTwoList
       ]);
     } else {
-      let insertObj = {};
+      let newPlayers = [];
```

Comment:
Maybe refactor this to play `playerArray` or something more descriptive? or `newPlayerArr`, either or will work for me, just to loosely define the types of objects that are being used and declared :)

Reply:
Done!